samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -189,9 +189,10 @@ For example, if you are using an nRF9160 DK version 1.0.1, use the following boa
 ``nrf9160dk@1.0.1/nrf9160/ns``
 
 This sample also supports placement of the MCUboot secondary partition in external flash for the nRF91x1 DKs, and for nRF9160 DK version 0.14.0 and higher.
-To enable this, add the following parameter to your build command:
+To enable this, add the following parameters to your build command:
 
-``-DEXTRA_CONF_FILE=overlay_mcuboot_ext_flash.conf``
+- ``-DEXTRA_CONF_FILE=overlay_mcuboot_ext_flash.conf``
+- ``-DSB_CONF_FILE=sysbuild_ext_flash.conf``
```

Comment:
```suggestion
* ``-DEXTRA_CONF_FILE=overlay_mcuboot_ext_flash.conf``
* ``-DSB_CONF_FILE=sysbuild_ext_flash.conf``
```
Hyphens works ok, but let's use asterisks for bullets, just for consistency.

Reply:
Sorry, I always forget this. I am so used to using dashes